CONTRIBUTING.md

```diff
@@ -50,54 +50,56 @@ Codespaces is new technology that allows you to use a container as your developm
 1. From the Azure SDK GitHub repo, click on the "Code -> Open with Codespaces" button.
 1. Open a Terminal. The development environment will be ready for you. Continue to [Building and Testing](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#building-and-testing).
 
-#### VS Code Codespaces
+### VS Code Codespaces
 
 1. Install the [VS Code Remote Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
 1. When you open the Azure SDK for C++ repo in VS Code, it will prompt you to open the project in the Dev Container. If it does not prompt you, then hit CTRL+P, and select "Remote-Containers: Open Folder in Container..."
 1. Open a Terminal. The development environment will be ready for you. Continue to [Building and Testing](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#building-and-testing).
 
-### Full Local Setup
+## Full Local Setup
 
-#### Pre-requisites
+### ** Pre-requisites **
 
-##### CMake
+#### -> CMake
 
 CMake version 3.13 or higher is required to build these libraries. Download and install CMake from the project's
 [website](https://cmake.org/download/).
 
-##### Third Party Dependencies
+### Third Party Dependencies
 
-- curl
-- libxml2
-- clang-format (min version 10)
+Azure SDK use Vcpkg manifest mode to declare the [list of required dependencies](https://github.com/Azure/azure-sdk-for-cpp/blob/main/vcpkg.json) for building the SDK service libraries. It will also get and set up Vcpkg automatically. You can move on to [Building the project](#building-the-project) and skip next part if you are not interested in learning the alternatives for setting up dependencies.
 
-Vcpkg can be used to install the Azure SDK for CPP dependencies into a specific folder on the system instead of globally installing them.
-Follow [vcpkg install guide](https://github.com/microsoft/vcpkg#getting-started) to get vcpkg and install the following dependencies:
+If you didn't jump to `Build the project`, you will now learn about how you can control Vcpkg and dependencies integration.
```

Comment:
nit: Trying to read through this, to understand what action I need to take and when. It seems a bit challenging to follow the flow of the doc, so snippets like these seem unnecessary.

This text could be the title of the next section.

Maybe remove this sentence.
```suggestion
```